src/main/java/frc/robot/utilities/Conversions.java

```diff
@@ -75,10 +71,6 @@ public static double magToDegrees(double magTicks, double gearRatio) {
         return magTicks / 4096f * 360 / gearRatio;
     }
 
-    public static double magToDegrees(double magTick) {
-        return magToDegrees(magTick, 1);
-    }
-
```

Comment:
why are you removing this?

Reply:
no default overloads here, if the gear ration is 1, put it when you call the function, as this will get cluttered very quickly